app/build.gradle.kts

```diff
@@ -97,10 +96,6 @@ android {
     }
 }
 
-composeCompiler {
-    featureFlags = setOf(ComposeFeatureFlag.StrongSkipping)
-}
-
```

Comment:
We get compiler warnings saying this is no longer necessary, as its on by default.